samples/kotlin_compose/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="Theme.CustomerioSDK" parent="android:Theme.Material.Light.NoActionBar" />
```

Comment:
I thought that Jetpack Compose ignores `<style>` themes? I thought that themes were defined via Compose? 

So, is this file needed?

Reply:
I plan to use [Splash screen](https://developer.android.com/develop/ui/views/launch/splash-screen), which requires `styles` for lower version APIs. 

Using this saves us from adding a custom new splash screen compasable and logic of how many X seconds it should stay etc.

Reply:
Do you believe that we need Splash screen? I want to avoid our sample apps having code that's irrelevant to QA or to customers using our SDK. Do you see a need for Splash screen and therefore this themes file?

Reply:
Splash is for the UX, these sample apps are not just going to be used by QA's but I think like before marketing team might use it to demo and test too. So having a black screen while the app loads seems like a bad experience to me and this is a very small tradeoff for it. 

And generally regarding 
> I want to avoid our sample apps having code that's irrelevant to QA or to customers

I think it varies, and I don't want all the sample apps to be same, because customers have different implementations themselves too. So its good if there are different approaches taken in different sample app and customers can relate to the one it finds closest to their implementation. 

We have had customers asking how they can use CIO using DI, so they won't have to replicate stuff etc, so the different variations don't hurt.